packages/combobox/src/Menu.styles.ts

```diff
@@ -0,0 +1,80 @@
+import { css } from '@leafygreen-ui/emotion';
+import { uiColors } from '@leafygreen-ui/palette';
+import { fontFamilies, typeScales, Mode } from '@leafygreen-ui/tokens';
+import { ComboboxSize } from './Combobox.types';
+import { menuItemHeight, menuItemPadding } from './constants';
+
+/**
+ * Menu styles
+ */
+
+export const popoverStyle = (width = 384) => css`
+  border-radius: 4px;
+  width: ${width}px;
+
+  & > * {
+    border-radius: inherit;
+  }
+`;
+
+export const menuBaseStyle = css`
+  position: relative;
+  width: 100%;
+  margin: 0;
+  padding: 0;
+  font-family: ${fontFamilies.default};
+  border-radius: inherit;
```

Comment:
Inherit rules are more brittle than explicit definitions. Is there a reason we can't use a definitive value here? We don't want this to change depending on context.

Reply:
Usually, that's handled through hiding overflow. Can we not do that here?